metrics/write.go

```diff
@@ -228,10 +255,13 @@ func (c *Client) write(ctx context.Context) error {
 				req := &prompb.WriteRequest{
 					Timeseries: tss[i:end],
 				}
-				if err := c.Store(context.TODO(), req); err != nil {
+
+				if _, err := c.remoteAPI.Write(ctx, req); err != nil {
 					merr.Add(err)
+					c.logger.Error("error writing metrics", "error", err)
```

Comment:
Should we log this downstream (when we handle merr?)

Reply:
I think we do, but this log is there to log the error as soon as it happens, which might be useful for detecting config issues early.